openmc/stats/univariate.py

```diff
@@ -1199,3 +1230,68 @@ def from_xml_element(cls, elem):
             distribution.append(Univariate.from_xml_element(pair.find("dist")))
 
         return cls(probability, distribution)
+
+    def integral(self):
+        """Return integral of the distribution
+
+        .. versionadded:: 0.13.1
+
+        Returns
+        -------
+        float
+            Integral of the distribution
+        """
+        return sum([
+            p*dist.integral()
+            for p, dist in zip(self.probability, self.distribution)
+        ])
+
+
+def combine_distributions(dists, probs):
+    """Combine distributions with specified probabilities
+
+    This function can be used to combine multiple instances of
+    :class:`~openmc.stats.Discrete` and `~openmc.stats.Tabular` into a single
```

Comment:
The `openmc.stats.Tabular` instances don't actually get combined into a single distribution in this code so we should remove it from the docstring or actually combine them. Although if they have different interpolation schemes and different x values that could be challenging.

Reply:
I've changed the language here to avoid "single distribution"